### ISSUE TITLE
outpainting result does not look good

### ISSUE THREAD

**Issue by xiyichen** [Insta360-Research-Team/DiT360#5]
Hi,
Thanks for releasing the code and models.
I tried to run the outpainting code in editing.py after fixing two bugs ("fenghora/DiT360-Panorama-Image-Generation" -> "Insta360-Research/DiT360-Panorama-Image-Generation", RFInversionParallelFluxPipeline.from_pretrained() -> RFPanoInversionParallelFluxPipeline.from_pretrained()). Using the demo example example/example.png with outpainting mask example/masks/mask0.png, the result does not look very good. Especially, the transition on the left boundary is not smooth:
<img width="2048" height="1024" alt="Image" src="https://github.com/user-attachments/assets/ae50cc45-a5dc-46e2-baf2-a70107d5e5db" />

I also tried to outpaint my own image, using the fov input and outpainting mask as below:
<img width="2048" height="1024" alt="Image" src="https://github.com/user-attachments/assets/029eb8a5-d964-42bf-89b8-f78f420b41c9" />
<img width="2048" height="1024" alt="Image" src="https://github.com/user-attachments/assets/824e2984-110d-4f1e-805b-8bfcbf1cf2bb" />
The result I got is:
<img width="2048" height="1024" alt="Image" src="https://github.com/user-attachments/assets/aee1dbde-3b26-4887-988d-b4a2c4810181" />.
No matter what prompt I use, it fails to outpaint a realistic outdoor scene.

Do you have any suggestions about how to get similar outpainting results as in your teaser image?
<img width="694" height="404" alt="Image" src="https://github.com/user-attachments/assets/8784820e-6be7-43a8-b28f-b87e53f083a9" />

 Thanks!

**Comment by fenghora** [comment]
Thank you very much for your detailed feedback and for taking the time to test our code!
Since our method was not specifically trained for inpainting or outpainting, some issues like unsmooth transitions can indeed occur.

We recommend adjusting the following parameters to improve the results:

- `tau` in [editing.py](https://github.com/Insta360-Research-Team/DiT360/blob/75e915a98a82615707421c53f1358a516fae6e1f/editing.py#L24)

- `t_l` in [pipeline.py](https://github.com/Insta360-Research-Team/DiT360/blob/75e915a98a82615707421c53f1358a516fae6e1f/pa_src/pipeline.py#L1053)

Please refer to the comments in the code and try tuning these values. Alternatively, you may also follow the FluxInpainting pipeline version for **fine-tuning**.
We plan to explore improvements in this area when our schedule allows.

Additionally, most of our training data are realistic images, while the example you provided looks more AI-rendered in style. This domain gap may also contribute to the suboptimal results.

**Comment by 3i-cao** [comment]
Hi, Did you face the problem of kernel get died when we run: pipe = RFPanoInversionParallelFluxPipeline.from_pretrained("black-forest-labs/FLUX.1-dev", torch_dtype=dtype).to(device)
I have 16GB VRAM available for loading
